zerver/openapi/zulip.yaml

```diff
@@ -5259,27 +5259,30 @@ paths:
         - name: narrow
           in: query
           description: |
-            The narrow where you want to fetch the messages from. See how to
+            The narrow filter where you want to fetch the messages from. See how to
             [construct a narrow](/api/construct-narrow).
 
-            Note that many narrows, including all that lack a `stream`
-            or `streams` operator, search the user's personal message
-            history. See [searching shared
-            history](/help/search-for-messages#searching-shared-history)
+            Note that many narrows, including all that lack a `stream` or `streams`
+            operator, search the user's personal message history. See [searching
+            shared history](/help/search-for-messages#searching-shared-history)
             for details.
 
-            For example, if you would like to fetch messages from all public streams instead
-            of only the user's message history, then a specific narrow for
-            messages sent to all public streams can be used:
+            For example, if you would like to fetch messages from all public
+            streams instead of only the user's message history, then a specific
+            narrow for messages sent to all public streams can be used:
             `{"operator": "streams", "operand": "public"}`.
 
-            Newly created bot users are not usually subscribed to any
-            streams, so bots using this API should either be
-            subscribed to appropriate streams or use a shared history
-            search narrow with this endpoint.
+            Newly created bot users are not usually subscribed to any streams,so
+            bots using this API should either be subscribed to appropriate streams
+            or use a shared history search narrow with this endpoint.
 
-            **Changes**: In Zulip 2.1.0, added support for using user/stream IDs
-            when constructing narrows for a message's sender, its stream and/or
+            **Changes**: In Zulip 7.0 (feature level 177), support was added for
+            three filters related to direct messages: `is:dm`, `dm` and `dm-including`;
```

Comment:
If you want to contextualize the term "filter" here, some options would be:

"three narrow filters"
(or "three narrow-filters", to avoid sounding like filters that are narrow)

"support in narrows was added for three filters"

"narrows gained support for three new filters"

Reply:
Thanks for these suggestions. I went with "narrows gained support for three new filters".